modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
```

Comment:
Can this be in a section called "Cluster and node preparation" or so

Reply:
The TOC for this content was restructured.